EIPS/eip-2228.md

```diff
@@ -20,22 +20,30 @@ This standard specifies the name for this network and provides reference example
 
 ## Motivation
 
-The Ethereum network with chainid 1 is referenced from authoritative sources using several names that are conflicting:
+The Ethereum network with chainid 1 is referenced using several conflicting names across EIPs (as published at the time EIP 2228 was written), client implementations, and information published on the internet at large:
 
-* Vitalik has called it "main net" at http://eips.ethereum.org/EIPS/eip-155 http://eips.ethereum.org/EIPS/eip-170
-* Vitalik has called it "mainnet" at http://eips.ethereum.org/EIPS/eip-86 
-* Vitalik has called it "Main net" at http://eips.ethereum.org/EIPS/eip-2* http://eips.ethereum.org/EIPS/eip-150
+* "main net" at [EIP-155](./eip-155.md), [EIP-170](./eip-170.md)
+* "mainnet" at [EIP-86 [DRAFT]](./eip-86.md), [client documentation](https://geth.ethereum.org/support/FAQ)
+* "Main net" at [EIP-2](./eip-2.md), [EIP-150](./eip-150.md)
    * \* In EIP-2, it is unclear if the first letter is capital because it is a proper noun or if it is the the first word in a text column presented in sentence case
-* Vitalik has called it "Mainnet" at http://eips.ethereum.org/EIPS/eip-7
-* Geth documentation calls its "mainnet" at https://geth.ethereum.org/support/FAQ
-* A final EIP here has referred to it as "Mainnet" at http://eips.ethereum.org/EIPS/eip-779
+* "Mainnet" at [EIP-7](./eip-7.md), [EIP-779](./eip-779.md)
+
+In several examples above, even papers written by the same author use (as published at the time EIP 2228 was written) inconsistent names to refer to the Ethereum network with chainid 1.
```

Comment:
```suggestion
The Ethereum network with chainid 1 is referenced using several conflicting names across EIPs, client implementations, and information published on the internet at large.  In several locations, even documents written by the same author use inconsistent names to refer to the Ethereum network with chainid 1.  Names in use at the time of this writing include:
* `main net`
* `mainnet`
* `Main net`
* `Mainnet`
```

At the least, you'll need to remove draft EIPs from the list and the external link to `https://geth.ethereum.org/support/FAQ`.  I don't believe that linking to these documents provides any future value to this EIP so I recommend just getting rid of all of the links and instead just providing a list of options you have seen used around the internet.

Also, I recommend adding something to the motivation section that explains *why* there is value in consistency here, as I think that is more important than showing that there is a historic lack of consistency.

Reply:
This is adopted substantially at 199121d174f28f35264980b19cfc96ec8fd8e8fd